shared/src/userData.ts

```diff
@@ -55,6 +56,7 @@ export interface Preferences {
   visibleSidebarCards: string[];
   returnToLink: string;
   isCalendarFullView: boolean;
+  isHideableRoadmapVisible: boolean;
```

Comment:
I think this name could be potentially confusing, because there's both the concept of it being "visible" ie "on the page" in some cases (for the up-and-running phase) as well as "visible" ie "open" when on the page. I think by this you mean the second one, so maybe we could avoid that confusion by naming it something like `isHideableRoadmapOpen` or something instead?